src/store.ts

```diff
@@ -1,88 +1,84 @@
-"use strict";
+'use strict';
 
-import path = require("path");
-import fs = require("fs");
-import child_process = require("child_process");
+import path = require('path');
+import fs = require('fs');
+import child_process = require('child_process');
 
-const userHome = require("user-home");
-const xdgBasedir = require("xdg-basedir");
+const userHome = require('user-home');
+const xdgBasedir = require('xdg-basedir');
 
 export function getHomeDirectory(): string {
-  return userHome;
+	return userHome;
 }
 
 export function getResourceDirectory(): string {
-  return path.join(xdgBasedir.data, "xpm") || path.join(getHomeDirectory(), ".xpm");
+	return path.join(xdgBasedir.data, 'xpm') || path.join(getHomeDirectory(), '.xpm');
 }
 
 export function getConfigDirectory(): string {
-  return path.join(xdgBasedir.config, "xpm") || path.join(getHomeDirectory(), ".xpm", "/config");
+	return path.join(xdgBasedir.config, 'xpm') || path.join(getHomeDirectory(), '.xpm', '/config');
 }
 
```

Comment:
Are these one-line functions necessary?  These seem like they could just be constants.

Also, I might consider a factory for doing things like this.